### DISCUSSION TITLE
Best way to include a Livewire component in every Filament response?

### DISCUSSION THREAD

**Discussion by myregistration** [filamentphp/filament#14139] [Help] [answered]
### Package

Other

### Package Version

v3

### How can we help you?

Since we aren't using Laravel's app.blade.php layout file when using Filament, what's the best way to include a @livewire component in every response?

**Comment by leandrocfe** [comment, marked as answer]
Maybe Render hooks?
https://filamentphp.com/docs/3.x/support/render-hooks#overview

**Reply by myregistration** [reply]
Yeah!  I've used that to include impersonate banner.  Didn't realize I could include livewire component that way.  Looks like it works well and in all my panels.  Thanks for the suggestion!
